parent/pom.xml

```diff
@@ -68,7 +68,7 @@
         <spoon.framework.version>8.4.0-beta-18</spoon.framework.version>
         <typesafe.version>1.4.1</typesafe.version>
         <hamcrest.version>2.2</hamcrest.version>
-        <scalecube.version>2.6.6</scalecube.version>
+        <scalecube.version>2.6.12</scalecube.version>
```

Comment:
While you're at it: could you please remove the `stopJmxMonitor` method from ScaleCubeClusterServiceFactory.java? :) This should be done by the scalecube itself in this version.

Reply:
I've already done that, that's why I updated the ScaleCube version in the first place

Reply:
Oops, sorry)